assets/dns/namespace.yaml

```diff
@@ -10,3 +10,6 @@ metadata:
     openshift.io/run-level: "0"
     # allow openshift-monitoring to look for ServiceMonitor objects in this namespace
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged
```

Comment:
Could you add a comment for these new labels?

Reply:
@candita fixed, ptal :-)